fontbe/src/glyphs.rs

```diff
@@ -331,6 +332,12 @@ impl Work<Context, AnyWorkId, Error> for GlyphWork {
             Error::GlyphError(ir_glyph.name.clone(), GlyphProblem::MissingDefault)
         })?;
 
+        // build a VariationModel specific to this glyph's master locations, upon which the region
+        // of influence and the delta weights associated to each master in turn depend.
+        let locations: HashSet<_> = ir_glyph.sources().keys().cloned().collect();
+        let var_model = VariationModel::new(locations, static_metadata.variable_axes.clone())
+            .map_err(|e| Error::VariationModelError(self.glyph_name.clone(), e))?;
```

Comment:
My intuition is that we would wind up building the same variation model over and over again, that if you made a set of all glyph instance locations it's size would be far lower than the number of glyphs.

Maybe that's best kicked into a follow-on PR given this does fix a major problem for us. I will file an issue to investigate.

Reply:
https://github.com/googlefonts/fontc/issues/415

Reply:
Checking if the glyph uses the same set of locations as the global variation model reduces the number of models created for Oswald from 884 to 1. #415 will generalize this but for now I'm inclined to sneak that in here, will push a commit.

Reply:
Oops, I merged and then realized I should have let you review my changes first. Sorry about that; if you see problems I am happy to fix them in a follow-on.